### ISSUE TITLE
Hull for arbitrary set

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/ValidatedNumerics.jl#129]
We need to add `hull` of an arbitrary set for IEEE 1788-2015 conformance.

**Comment by lbenet** [comment]
Can you be more specific? To which section do you refer?

**Comment by dpsanders** [comment]
Section 10.3 defines the hull of an arbitrary set. It's not clear if that's actually required, but it is not too difficult to add for a finite set of points (and that may be a useful operation for certain things). @oheim may have an opinion here.

**Comment by oheim** [comment]
The hull operation in Section 10.3 is only a mathematical definition and not required to be implemented.

Section 9.3 (or 10.5.7) requires to implement ``convexHull``, which is an operation in two **intervals**.

In Octave, I have implemented [a hull function](http://octave.sourceforge.net/interval/function/hull.html) which accepts a variable number of parameters (varargs). However, I did this because I find it useful and it behaves different compared to [convexHull (union in Octave)](http://octave.sourceforge.net/interval/function/@infsup/union.html) regarding decorations.
